samples/manual-unsubscribe/sample.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Manual unsubscribe
+summary: Shows how to manually remove subscriptions when subscribers are decommissioned.
+reviewed: 2017-07-10
+component: Core
+related:
+- transports
+- transports/msmq
+- transports/sql
+- transports/azure-storage-queues
+- transports/sqs
+---
+
+This sample shows how to manually remove subscriptions when subscribers are decommissioned. The solution comprises of 4 projects.
+
+NOTE: While this sample uses the [MSMQ transport](/transports/msmq), the concepts shown are valid for all transports based on message driven subscriptions and that don't support native pub/sub. For more information see [Publish-Subscribe](/nservicebus/messaging/publish-subscribe/).
+
+
+## Running the sample
+
+
+### Initial State
+
+Run the sample and all three endpoints will start.
+
+Subscriber initially subscribes to the `SomethingHappened` message in Publisher.
+
+
+### Publish event
+
+Hit `enter` in Publisher and a `SomethingHappened` will be published via the following process:
+
+```mermaid
+sequenceDiagram
+Participant Subscriber
+Participant Publisher
+Participant Persistence
+Note over Publisher: Publish SomethingHappened
+Publisher ->> Persistence: Requests "who wants SomethingHappened"
+Persistence ->> Publisher: "Subscribe"
+Publisher ->> Subscriber: Send SomethingHappened
+```
+
+The `SomethingHappened` event will be received by Subscriber.
+
+
+### Unsubscribe
+
+Hit enter on SubscriptionManager. A `ManualUnsubscribe` message for `SomethingHappened` will be sent to Publisher.
+
+```mermaid
+sequenceDiagram
+Participant SubscriptionManager AS SubscriptionManager
+Participant Publisher
+Participant Persistence
+SubscriptionManager ->> Publisher: Send unsubscribe "SomethingHappened"
+Publisher ->> Persistence: Store "unsubscribe from SomethingHappened"
+Note over Publisher: Publish SomethingHappened
+Publisher ->> Persistence: Requests "who wants SomethingHappened"
+Persistence ->> Publisher: "No Endpoints"
+Note over Publisher: No Send
+```
+
+Now hit `enter` in Publisher and no `SomethingHappened` will be published.
+
+
+## Solution structure
+
+
+### Subscriber
+
+A sample endpoint subscribed to an event that will be published by `Publisher`:
+
+snippet: event-handler
+
+
+### Publisher
+
+
+#### The publisher configuration
+
+snippet: publisher-config
+
+This specific sample is configured to use [MSMQ Subscription Persistence](/persistence/msmq/subscription.md).
+
+
+#### Subscriber decommissioning
+
+In such a configuration, when a subscriber endpoint is decommissioned it may happen that subscriptions remain stored at the publisher. Publishers have no way to detect that a subscriber is no longer available. They will continue to publish events even for subscribers that no longer exist. This behavior will eventually lead to storage and quota issues.
```

Comment:
@mikeminutillo how is this? https://github.com/Particular/docs.particular.net/pull/2907/commits/7f281dd2ecd2a16dc00bfb5baf9bbfe3c50b1032